content/docs/developing-operators/getting-started.md

```diff
@@ -30,15 +30,7 @@ This is a pretty normal Kubernetes YAML file defining a deployment. However, you
 
 <<< @/kudo/test/integration/first-operator-test/first-operator/params.yaml
```

Comment:
I think this section needs some more love. I agree that we should 't repeat ourselves, however:
- we jump directly into the yaml files, without even explaining *what operator we're building* and what should be expected at the end
- the _"Details for the concepts"_ part is important but should probably be placed at the beginning as a "Prerequisite" to the rest of it